Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -788,6 +788,10 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
         WebProcess::singleton().switchFromStaticFontRegistryToUserFontRegistry(WTFMove(parameters.fontMachExtensionHandles));
 #endif
 
+#if HAVE(HOSTED_CORE_ANIMATION)
+    WebProcess::singleton().setCompositingRenderServerPort(WTFMove(parameters.acceleratedCompositingPort));
```

Comment:
Is this correct? Technically a web process can have multiple web pages and the pages can have different preferences, so WebProcess should set this port as long as one page sets it. It seems existing implementation is the page added later can overwrite the setting of previous page.

Reply:
That's a good point, will fix!

Thanks for reviewing!